node/src/components/consensus/era_supervisor/era.rs

```diff
@@ -126,6 +126,8 @@ impl PendingCandidate {
 pub struct Era<I> {
     /// The consensus protocol instance.
     pub(crate) consensus: Box<dyn ConsensusProtocol<I, ClContext>>,
+    /// The scheduled starting time of this era.
+    pub(crate) start_time: Timestamp,
```

Comment:
It's also part of `HighwayProtocol.highway.state.params` right? We probably have to make sure that they never diverge.

Reply:
I think this is actually passed in here from the params. The only change is we're saving it in the era metadata for later.